addons/addon-base-raas/packages/base-raas-workflow-steps/lib/plugins/workflow-steps-plugin.js

```diff
@@ -30,6 +30,10 @@ const stopEC2Environment = require('../steps/stop-ec2-environment/stop-ec2-envir
 const stopEC2EnvironmentYaml = require('../steps/stop-ec2-environment/stop-ec2-environment.yml');
 const networkInfra = require('../steps/storage-gateway/create-network-infrastructure');
 const networkInfraYaml = require('../steps/storage-gateway/create-network-infrastructure.yml');
+const bulkReachabilityInfra = require('../steps/bulk-reachability-check/bulk-reachability-check');
```

Comment:
Any reason why you are calling it bulkReachability**Infra** ,  why 'Infra'?

Reply:
Fixed